app/views/home/_vacancy_facets.html.slim

```diff
@@ -15,7 +15,7 @@
         ul.govuk-list.two-column-list aria-label="#{t(".counties")} list"
           - @vacancy_facets.counties.each do |county, count|
             li
-              = govuk_link_to location_path(county) do
+              = govuk_link_to location_path(county).parameterize do
```

Comment:
It would be better to just call `parameterize` on `county`, not the whole path (here and in the other places in this file), because that's the only bit that needs it.